db/migrate/20181105212016_duplicate_deleted_at_state_changed_at.rb

```diff
@@ -2,11 +2,12 @@
 
 class DuplicateDeletedAtStateChangedAt < ActiveRecord::Migration
   def up
-    Account.transaction do
-      Account.scheduled_for_deletion.where.has{ deleted_at != nil }.find_each do |account|
-        account.update_column(:state_changed_at, account.deleted_at) or raise ActiveRecord::Error
-      end
-    end
+    # Done in a query instead of Ruby because later on deleted_at is removed programatically and otherwise it says it doesn't exist
+    query = <<~SQL
+      UPDATE accounts
```

Comment:
I think Oracle will complain
You can use activerecord queries `where `+ `update_all`

Reply:
@hallelujah Oracle didn't complain because I tested it locally 😄

Reply:
But it is still better to use ruby I think